.github/workflows/_build.yml

```diff
@@ -115,7 +110,6 @@ jobs:
       - name: Android Tests
         uses: reactivecircus/android-emulator-runner@v2
         with:
-          target: google_apis
           profile: Nexus 4
```

Comment:
Not part of this but do we need to change the profile to a newer device given the Nexus 4 even in physical device is no longer supported?

Reply:
good question. it doesn't _really_ matter what device we're using - the OS version is the important part. everything else doesn't really change much aside from resolution etc which is a wild west anyways